doc/manual/performance-tips.rst

```diff
@@ -1114,6 +1114,53 @@ properties:
    LLVM auto-vectorization may kick in automatically, leading to no further
    speedup with :obj:`@simd`.
 
+While :obj:`@simd` needs to be placed in front of a loop, that is not the case
+for :obj:`@inbounds`. It can be placed in front of any expression as long as it
+is inside a function. The only place where it can be outside of a function
+definition is in front of a loop.
```

Comment:
Well, it is also just not true. At global scope it is perfectly fine to write:

```julia
x = 1
@inbounds x
```
But whereas without `@inbounds` this would print `1`, with `@inbounds` it doesn't print anything. However, I think the inbounds meta information is ignored outside of a function.

I would simplify the description to say that `@inbounds` can be written in front of any expression, and in some expressions this may be useful to limit the scope of what you are declaring to be "in bounds".